Sources/Apollo/GraphQLExecutor.swift

```diff
@@ -245,6 +245,7 @@ final class GraphQLExecutor<FieldCollector: FieldSelectionCollector> {
   ) throws -> FieldSelectionGrouping {
     var grouping = FieldSelectionGrouping(info: info)
 
+    #warning("TODO: When executing on a selection set that isn't operation root. This will be wrong.")
```

Comment:
Will this be handled in a subsequent PR? I think that's what we discussed in our last conversation.

Reply:
Yupp!